tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ConfigurationTest.java

```diff
@@ -47,6 +49,22 @@ public void testStartWithDuplicateQueues() throws Exception {
       }
    }
 
+   @Test
+   public void testQueueWithoutAddressName() throws Exception {
+      final SimpleString QUEUE_NAME = RandomUtil.randomSimpleString();
+      ActiveMQServer server = createServer(false, createDefaultInVMConfig());
+      try {
+         server.getConfiguration().addQueueConfiguration(new CoreQueueConfiguration().setName(QUEUE_NAME.toString()));
```

Comment:
how would that look like on the xml?

Reply:
I don't think this would be possible via the XML. It's only possible by using the `CoreQueueConfiguration` object directly.